cloudslang_dsl_reference.rst

```diff
@@ -853,7 +852,9 @@ input name may in turn be mapped to its properties or an input
 `expression <#expressions>`__.
 
 Inputs are used to pass parameters to `flows <#flow>`__ or
-`operations <#operation>`__.
+`operations <#operation>`__. Input names for a specific `flow <#flow>`__ or
+`operation <#operation>`__ must be different than the `output <#outputs>`__
+names of the same `flow <#flow>`__ or `operation <#operation>`__.
```

Comment:
would also write about the fact that step inputs should be named different than the outputs of the flow/operation used in that step